styles/Home.module.css

```diff
@@ -48,12 +64,28 @@
   border-bottom: 0;
 }
 
-.applicant div {
-  flex: 1 1 50%;
+.applicant div,
+.table-heading div {
+  flex: 1 1;
   padding: 10px;
   border-right: 1px solid black;
 }
 
-.applicant div:last-child {
+.applicant div:last-child,
+.table-heading div:last-child {
   border-right: 0;
 }
+
+.pagination {
+  display: flex;
+  justify-content: center;
+  margin-top: 20px;
+}
+
+.pagination button {
+  margin: 0 10px;
+}
+
+.border-bottom-none {
```

Comment:
This stops the borders of the table and table headers from overlapping when there are no results:

<img width="776" alt="border-bug" src="https://github.com/user-attachments/assets/2d5afe28-127a-4743-84ac-4d9c5ade6be3">